sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppNavigation.kt

```diff
@@ -43,9 +54,29 @@ private fun NavGraphBuilder.composable(
     )
 }
 
+/**
+ * Navigate to [AppDestination] with navArgument
+ */
 private fun NavHostController.navigate(appDestination: AppDestination) {
     when (appDestination) {
         is AppDestination.Up -> navigateUp()
         else -> navigate(route = appDestination.destination)
     }
 }
+
+/**
+ * Navigate to AppDestination with Parcelable Data
+ * Caution to use this method. This method use savedStateHandle to store the Parcelable data.
+ * When previousBackstackEntry is popped out from navigation stack, savedStateHandle will return null and cannot retrieve data.
+ * eg.Login -> Home, the Login screen will be popped from the back-stack on logging in successfully.
+ * Navigate to provided [AppDestination] with a Pair of key value String and Data [parcel]
+ */
```

Comment:
Seems like they are implying the same message 🙏 
```suggestion
/**
 * Navigate to provided [AppDestination] with a Pair of key value String and Data [parcel]
 * Caution to use this method. This method use savedStateHandle to store the Parcelable data.
 * When previousBackstackEntry is popped out from navigation stack, savedStateHandle will return null and cannot retrieve data.
 * eg.Login -> Home, the Login screen will be popped from the back-stack on logging in successfully.
 */
```